CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ All notable changes to this project will be documented in this file.
 
 -   #1587 : Fix unnecessarily long file names generated by `epyccel`.
 
+### Added
+
+-   #1571 : Add support for the function `tuple`.
```

Comment:
There is already an "Added" section above. Please can you move the bullet point to the existing section